server/server.js

```diff
@@ -117,4 +119,13 @@ app.get('/questions', (req, res) => {
     });
 });
 
+app.post('/interactions', (req, res) => {
+  atelierQueries.postInteraction(req.body, atelierHeaders)
```

Comment:
Putting all of the queries in a separate module was an excellent design decision. Always super clean work!